apps/storefront-e2e/src/support/mixins/page-with-product-list.mixin.ts

```diff
@@ -24,11 +24,11 @@ export function WithProductList<TPage extends Constructor<E2EPage>>(
       cy.intercept('/catalog-search?**').as('catalogSearch');
     }
 
-    waitForSearchRequest(): void {
-      cy.wait('@catalogSearch');
+    waitForSearchRequest(skipRequest?: boolean): void {
+      if (!skipRequest) cy.wait('@catalogSearch');
```

Comment:
Let's create a separate method like `skipSearchRequest()` or `waitForProductsToLoad()` in this page object, because `waitForSearchRequest(true)` looks weird and is confusing